scripts/makeNativeHeaders.ts

```diff
@@ -2,18 +2,26 @@ import fs from 'fs'
 import path from 'path'
 
 function makeNativeHeaders() {
+  // Grab the API key:
   let apiKey = 'Error: Set up env.json & re-run scrips/makeNativeHeaders.js'
   try {
     apiKey = require('../env.json').AIRBITZ_API_KEY
   } catch (e) {
     console.log(apiKey)
   }
 
+  // Grab the push notification server:
+  let pushServer = 'https://push2.edge.app'
```

Comment:
Do not allow push2.edge.app to be hard coded. We want to ensure we only use the push server specified in the appConfig. Fallback should just be no push servers.